common/src/Microsoft.Azure.IIoT.AspNetCore.OpenApi/src/Extensions/ApplicationBuilderEx.cs

```diff
@@ -32,8 +32,6 @@ public static class ApplicationBuilderEx {
             var addresses = app.ServerFeatures.Get<IServerAddressesFeature>()?.Addresses
                 .Select(a => new Uri(a.Replace("://*", "://localhost")))
                 .ToList() ?? new List<Uri>();
-            var path = addresses.FirstOrDefault()?.PathAndQuery.Trim('/') ??
```

Comment:
Removing this will break the services all container swagger I think.

Reply:
I've tested this for Services.All and it seems to work fine. Swaggers of all components that I've checked came up and showed correct endpoints.

The only problem that I saw there was that a few Swagger pages were empty (there were no endpoint definitions). This happened for the following components:

* configuration
* edge/manage

Is this expected ?

Reply:
@marcschier, the address of `openapi.json` has not changed. It is still served under `/swagger/v2/openapi.json` or `/<component_name>/swagger/v2/openapi.json`.

The difference is that `options.SwaggerEndpoint($"{info.Version}/openapi.json", info.Version)` now specifies it to be on `v2/openapi.json` endpoint **relative** to Swagger UI. Previously we were specifying an **absolute** endpoint, since it was starting with `/`. So do you still think that `swagger.cmd` has to be changed ?

Reply:
No - sounds great 👍